[pulsar-client] NPE while call getLastMessageId

**Describe the bug**
Meet NPE while call getLastMessageId as below:
```
[root@xxx apache-pulsar-2.6.0-SNAPSHOT]# bin/pulsar-admin topics last-message-id persistent://public/default/test
java.lang.RuntimeException: com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: org.apache.pulsar.client.impl.BatchMessageIdImpl["acker"]->org.apache.pulsar.client.impl.BatchMessageAcker["outstandingAcks"])
        at org.apache.pulsar.admin.cli.CliCommand.print(CliCommand.java:182)
        at org.apache.pulsar.admin.cli.CmdTopics$GetLastMessageId.run(CmdTopics.java:773)
        at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:71)
        at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:221)
        at org.apache.pulsar.admin.cli.PulsarAdminTool.main(PulsarAdminTool.java:263)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: org.apache.pulsar.client.impl.BatchMessageIdImpl["acker"]->org.apache.pulsar.client.impl.BatchMessageAcker["outstandingAcks"])
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:394)
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:353)
        at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:316)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:730)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:166)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:727)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:722)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:166)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
        at com.fasterxml.jackson.databind.ObjectWriter$Prefetch.serialize(ObjectWriter.java:1433)
        at com.fasterxml.jackson.databind.ObjectWriter._configAndWriteValue(ObjectWriter.java:1135)
        at com.fasterxml.jackson.databind.ObjectWriter.writeValueAsString(ObjectWriter.java:1005)
        at org.apache.pulsar.admin.cli.CliCommand.print(CliCommand.java:180)
        ... 4 more
Caused by: java.lang.NullPointerException
        at org.apache.pulsar.client.impl.BatchMessageAcker.getOutstandingAcks(BatchMessageAcker.java:69)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:688)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:722)
        ... 14 more
```
The reason is that `BatchMessageIdImpl` use a default `BatchMessageAcker` with null value.

**Expected behavior**
`getLastMessageId` should be successful like below:
```
[root@xxx apache-pulsar-2.6.0-SNAPSHOT]# bin/pulsar-admin topics last-message-id persistent://public/default/test
{
  "ledgerId" : 5,
  "entryId" : 0,
  "partitionIndex" : -1,
  "batchIndex" : 9,
  "acker" : {
    "batchSize" : 0,
    "prevBatchCumulativelyAcked" : false,
    "outstandingAcks" : 0
  },
  "outstandingAcksInSameBatch" : 0,
  "batchSize" : 0
}

```

--- a/pulsar-client/src/main/java/org/apache/pulsar/client/impl/BatchMessageAcker.java
+++ b/pulsar-client/src/main/java/org/apache/pulsar/client/impl/BatchMessageAcker.java
@@ -24,7 +24,7 @@ import java.util.BitSet;
 class BatchMessageAcker {
 
     private BatchMessageAcker() {
-        this.bitSet = null;
+        this.bitSet = new BitSet();
         this.batchSize = 0;
     }
 

--- a/pulsar-client/src/main/java/org/apache/pulsar/client/impl/BatchMessageAckerDisabled.java
+++ b/pulsar-client/src/main/java/org/apache/pulsar/client/impl/BatchMessageAckerDisabled.java
@@ -18,12 +18,14 @@
  */
 package org.apache.pulsar.client.impl;
 
+import java.util.BitSet;
+
 class BatchMessageAckerDisabled extends BatchMessageAcker {
 
     static final BatchMessageAckerDisabled INSTANCE = new BatchMessageAckerDisabled();
 
     private BatchMessageAckerDisabled() {
-        super(null, 0);
+        super(new BitSet(), 0);
     }
 
     @Override

--- a/pulsar-client/src/test/java/org/apache/pulsar/client/impl/BatchMessageIdImplTest.java
+++ b/pulsar-client/src/test/java/org/apache/pulsar/client/impl/BatchMessageIdImplTest.java
@@ -21,7 +21,11 @@ package org.apache.pulsar.client.impl;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNotEquals;
 import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import org.apache.pulsar.common.util.ObjectMapperFactory;
 import org.testng.annotations.Test;
 
 public class BatchMessageIdImplTest {
@@ -66,6 +70,32 @@ public class BatchMessageIdImplTest {
         assertEquals(batchMsgId4, msgId);
 
         assertEquals(msgId, batchMsgId4);
+    }
+
+    @Test
+    public void deserializationTest() {
+        // initialize BitSet with null
+        BatchMessageAcker ackerDisabled = new BatchMessageAcker(null, 0);
+        BatchMessageIdImpl batchMsgId = new BatchMessageIdImpl(0, 0, 0, 0, ackerDisabled);
+
+        ObjectWriter writer = ObjectMapperFactory.create().writerWithDefaultPrettyPrinter();
+
+        try {
+            writer.writeValueAsString(batchMsgId);
+            fail("Shouldn't be deserialized");
+        } catch (JsonProcessingException e) {
+            // expected
+            assertTrue(e.getCause() instanceof NullPointerException);
+        }
+
+        // use the default BatchMessageAckerDisabled
+        BatchMessageIdImpl batchMsgIdToDeserialize = new BatchMessageIdImpl(0, 0, 0, 0);
+
+        try {
+            writer.writeValueAsString(batchMsgIdToDeserialize);
+        } catch (JsonProcessingException e) {
+            fail("Should be successful");
+        }
     }
 
 }

